2013/pagrindinis/u1/u1.py

```diff
@@ -12,13 +12,15 @@ def read_row(lines):
     x, y = [abs(int(x)) for x in line[10:].split()]
```

Comment:
pataisiau (ir dar truputį..)

2015-04-13 10:22 GMT+03:00 sirex <notifications@github.com>:

> In 2013/pagrindinis/u1/u1.py
> <https://github.com/python-dirbtuves/it-brandos-egzaminai/pull/7#discussion_r28218669>
> :
>
> > @@ -12,13 +12,15 @@ def read_row(lines):
> >      x, y = [abs(int(x)) for x in line[10:].split()]
>
> Taip, reikėtų abs iškelti iš duomenų nuskaitymo dalies. Ar pataisysi tai?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/python-dirbtuves/it-brandos-egzaminai/pull/7/files#r28218669>
> .
>



-- 
Jurgis Pralgauskis
tel: 8-616 77613;
Don't worry, be happy and make things better ;)
http://galvosukykla.lt